src/main/scala/it/pagopa/pdnd/interop/uservice/partymanagement/error/RelationshipAlreadyExists.scala

```diff
@@ -0,0 +1,15 @@
+package it.pagopa.pdnd.interop.uservice.partymanagement.error
+
+import it.pagopa.pdnd.interop.uservice.partymanagement.model.RelationshipProductSeed
+import it.pagopa.pdnd.interop.uservice.partymanagement.model.party.PersistedPartyRole
+
+import java.util.UUID
+
+final case class RelationshipAlreadyExists(
```

Comment:
what if we use only the relationship id?

Reply:
The relationship id is spawned when we receive the request, so it is always different.

Reply:
probably I'm missing something here, but this exception states that a relationship already exists, so shouldn't we have a specific relationship id at this point?